src/monster.cpp

```diff
@@ -932,7 +926,7 @@ void Monster::onThinkDefense(uint32_t interval)
 		}
 	}
 
-	if (!isSummon() && summons.size() < mType->maxSummons) {
+	if (!isSummon() && summons.size() < mType->maxSummons && hasFollowPath) {
```

Comment:
@marksamman Could you help us what do you want test? Also, I can test it.
I want see if there is any side effects of theses changes.